cascade-api/src/Cascade/Api/Database/Project.hs

```diff
@@ -40,11 +40,14 @@ instance Table ProjectTable where
   newtype PrimaryKey ProjectTable f = PrimaryKey
     { unPrimaryKey :: Wrapped.C f Project.Id
     }
-    deriving stock Generic
+    deriving stock (Generic)
     deriving anyclass Beamable
   primaryKey = PrimaryKey . id
 
 type Row = ProjectTable Identity
 
 deriving stock instance Show Row
 deriving stock instance Eq Row
+
+deriving stock instance Show (PrimaryKey ProjectTable Identity)
+deriving stock instance Eq (PrimaryKey ProjectTable Identity)
```

Comment:
It's better to keep standalone instances as near as possible to the target datatype. So as a convention I suggest moving these standalone instance deriving lines to line 46.